js/apps/system/_admin/aardvark/APP/frontend/js/views/indicesView.js

```diff
@@ -366,10 +366,10 @@
 
     selectIndexType: function () {
       $('.newIndexClass').hide();
-      var type = $('#newIndexType').val();
+      var type = arangoHelper.escapeHtml($('#newIndexType').val());
       if (type === null) {
         type = $('#newIndexType').children().first().attr('value');
-        $('#newIndexType').val(type);
+        arangoHelper.escapeHtml($('#newIndexType').val(type));
```

Comment:
seems wrong. shouldn't it be `$('#newIndexType').val(arangoHelper.escapeHtml(type));`?